src/core/config/mac.h

```diff
@@ -489,16 +489,25 @@
 #endif
 
 /**
- * @def OPENTHREAD_CONFIG_CSL_MIN_RECEIVE_ON
```

Comment:
Let's add a check for the removed/renamed old config in `openthread-core-config-check.h` (so that if any platform/product happens to use the old config, gets a build error and updates accordingly).

Reply:
I already added the check, maybe you missed it? :)

Reply:
Actually I had removed the previous `#endif`, now it's fixed.